env.sh

```diff
@@ -15,8 +15,7 @@
 export X_CSI_VXFLEXOS_THICKPROVISION=false
 export X_CSI_VXFLEXOS_ENABLESNAPSHOTCGDELETE="true"
 export X_CSI_VXFLEXOS_ENABLELISTVOLUMESNAPSHOTS="true"
-#Uncomment X_CSI_QUOTA_ENABLED if you want to enable quota for nfs volume
-#export X_CSI_QUOTA_ENABLED="true"
+export X_CSI_QUOTA_ENABLED="true"
```

Comment:
Why are we removing the comment here for X_CSI_QUOTA_ENABLED usage?

Reply:
Quota will be set to true by default , so we don't need to comment it